webpack.config.js

```diff
@@ -63,18 +64,47 @@ module.exports = {
         ],
       },
       {
-        test: /\.svg$/,
-        loader: 'file-loader',
-        query: {
-          name: 'static/media/[name].[hash:8].[ext]',
-        },
-      },
-      {
-        test: /\.(woff|woff2|eot)$/,
+        test: /\.(woff|woff2|eot|png|jpe?g|gif)$/,
         use: {
           loader: 'url-loader',
         },
       },
+      {
+        test: /\.svg$/,
+        include: [/src\/svgs\//],
+        use: [
+          {
+            loader: 'svg-sprite-loader',
+          },
+          {
+            loader: 'svgo-loader',
+            options: {
+              plugins: [
+                { removeTitle: true },
+                { removeNonInheritableGroupAttrs: true },
+                { removeUselessStrokeAndFill: false },
```

Comment:
The idea here is you opt in for removal of any presentational attributes with the idea that you'll use external CSS styles to account for these

Reply:
Maybe we can add documentation on the styleguide that includes what attributes are available via CSS (didn't see that `removeTitle` was checked as "True" -- wouldn't we want to default to include the title for accessibility purposes?

Reply:
Ahh yes, so removing the title is something that folks do for large sets of icons that are  meant to be decorative (https://dev.to/adammcquiff/how-and-why-to-clean-svg-markup-49i shows a bit about that), but, since we're sort of straddling the lines likely, I'll remove this rule so it'll default to false. If we leave it as false, and start to add a bunch of icons, we'll either want to remove titles later, or, ensure we at least provide useful titles as they'll get announced by screen readers

Reply:
But good call! So, I'm about to push up a commit now that leaves <title> tags intact:

![image](https://user-images.githubusercontent.com/142403/70001732-e7c0fa80-1512-11ea-9d27-52c3ea767d86.png)